docs/toc.md

```diff
@@ -167,6 +167,7 @@
 # Developer tools
 ## [AlignmentGrid](developer-tools/AlignmentGrid.md)
 ## [FocusTracker](developer-tools/FocusTracker.md)
+## [Guard](developer-tools/Guard.md)

```

Comment:
Does it make better sense for us to have a whole .NET section to the docs as the 'developer tools' are currently all UWP helper controls?

Reply:
Then we could move the Array Extensions docs in there and maybe others too?